packages/experimental/Menu/src/Menu.tsx

```diff
@@ -0,0 +1,14 @@
+/** @jsx withSlots */
+import { menuName, MenuProps, MenuTokens } from './Menu.types';
+import { compressible, buildUseTokens, UseTokens } from '@fluentui-react-native/framework';
+
+const useMenuTokens = buildUseTokens<MenuTokens>(() => ({}), menuName);
+
+export const Menu = compressible<MenuProps, MenuTokens>((_props: MenuProps, _useTokens: UseTokens<MenuTokens>) => {
```

Comment:
Just curious, did you choose compressible just because it was simpler to create a skeleton of, or some other reason?

Reply:
I switched to compressible since looking at the spec for the outer menu component, it didn't seem like it needed all of the stuff from compose.

In fact locally I've "downgraded" it even further to stagedComponent